packages/core/src/Interweave.tsx

```diff
@@ -13,9 +13,10 @@ export default function Interweave(props: InterweaveProps) {
     matchers = [],
     onAfterParse = null,
     onBeforeParse = null,
-    tagName = 'span',
+    wrap = true,
```

Comment:
This should be the inverse, `noWrap`, as writing `wrap={false}` is ugly.

Reply:
Yeah, makes sense. Will change.